devices/common_patches/h7_exti_dualcore.yaml

```diff
@@ -0,0 +1,853 @@
+          bitWidth: 1
+        MR65:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 01
+          bitWidth: 1
+        MR66:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 02
+          bitWidth: 1
+        MR67:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 03
+          bitWidth: 1
+        MR68:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 04
+          bitWidth: 1
+        MR69:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 05
+          bitWidth: 1
+        MR70:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 06
+          bitWidth: 1
+        MR71:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 07
+          bitWidth: 1
+        MR72:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 08
+          bitWidth: 1
+        MR73:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 09
+          bitWidth: 1
+        MR74:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 10
+          bitWidth: 1
+        MR75:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 11
+          bitWidth: 1
+        MR76:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 12
+          bitWidth: 1
+        MR77:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 13
+          bitWidth: 1
+        MR78:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 14
+          bitWidth: 1
+        MR79:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 15
+          bitWidth: 1
+        MR80:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 16
+          bitWidth: 1
+        MR82:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 18
+          bitWidth: 1
+        MR84:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 20
+          bitWidth: 1
+        MR85:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 21
+          bitWidth: 1
+        MR86:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 22
+          bitWidth: 1
+        MR87:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 23
+          bitWidth: 1
+        MR88:
+          description: CPU2 interrupt Mask on Direct Event input x+64
+          bitOffset: 24
+          bitWidth: 1
+      C1EMR3:
```

Comment:
Wrong indentation, and should be C2EMR3?

Reply:
Fixed, along with register names